pkg/vetter/conflictingvirtualservicehost/vet_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("Conflicting Virtual Service Host Vet Notes", func() {
 			ObjectMeta: metav1.ObjectMeta{
 				Name:         "Vs4",
 				Namespace:    namespace,
-				Initializers: &metav1.Initializers{}},
```

Comment:
what's this change?

Reply:
It wasn't compiling with it there (`Initializers` must no longer be a member of `ObjectMeta`). Removed it and the tests still pass.